lib/config/PluginSettings.ts

```diff
@@ -11,15 +11,31 @@ export interface PluginSettings {
    */
   workspace: TogglWorkspace;
 
+  /** Has dismissed the update alert for 0.4.0 */
+  hasDismissedAlert?: boolean;
+  
+  /** Update the day's total time in real-time in the sidebar */
+  updateInRealTime?: boolean;
+
   /**
    * The max. allowed characters in the title of a timer in
    * the status bar.
    */
   charLimitStatusBar: number;
 
-  /** Has dismissed the update alert for 0.4.0 */
-  hasDismissedAlert?: boolean;
+  /**
+   * The time format for the status bar.
+   */
+  statusBarFormat?: string;
 
-  /** Update the day's total time in real-time in the sidebar */
-  updateInRealTime?: boolean;
+  /**
+   * The prefix to show before the time entry in the status bar.
+   */
+  statusBarPrefix?: string;
+
+  /** Whether to show the project in the status bar. */
+  statusBarShowProject?: boolean;
+
+  /** Message shown in the status bar when no time entry is running. */
+  statusBarNoEntryMesssage?: string;
```

Comment:
Personally I don't see a need to customise something like this, can you elaborate on your use case?

Reply:
Default setting: "-"
I currently have it set to "No time entry running"; I can imagine others may prefer a different message or for it to be completely blank (hiding the status bar entry when no time entry is running).